user-interface/test/accessibility/case-notes.test.ts

```diff
@@ -0,0 +1,32 @@
+import test, { expect } from '@playwright/test';
+import AxeBuilder from '@axe-core/playwright';
+
+test.describe('Case Notes', () => {
+  test.describe.configure({ retries: 0, mode: 'serial' });
+
+  let addCaseNoteButton;
+
+  test.beforeEach(async ({ page }) => {
+    await page.goto(`http://localhost:3000/case-detail/999-99-00001/notes`);
+
+    addCaseNoteButton = page.getByTestId('open-modal-button_case-note-add-button');
+    await expect(addCaseNoteButton).toBeVisible();
+  });
+
+  test('should not have accessibility issues', async ({ page }) => {
+    const testNoteTitle = 'Test Note Title';
+    const testNoteContent = 'Test Note Content for E2E purposes';
+
```

Comment:
Should we add an axe check here before we open the modal? Also, we may want to find a way to have there be a draft note and maybe even a draft edit so that the alerts are included in the test.